ai_economist/foundation/components/redistribution.py

```diff
@@ -357,6 +404,9 @@ def curr_marginal_rates(self):
                 np.array(self.us_federal_single_filer_2018_scaled), self.curr_rate_max
             )
 
+        if self.tax_model == "saez":
+            return np.minimum(self.curr_bracket_tax_rates, self.curr_rate_max)
+
         if self.tax_model == "fixed-bracket-rates":
             return np.minimum(np.array(self.fixed_bracket_rates), self.curr_rate_max)
 
```

Comment:
else clause: what if the tax_model is not implemented?

Reply:
same reason as above - else not necessary